### PR TITLE
Update examples to use Sighthound test cameras

### PR DESCRIPTION
**Pull request by cpeskin12**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,44 +45,44 @@ Run `python3 data-apy.py --help` for an overview.
 ### Examples:
 Query data for collision sensor on BAI_000646 for the last 3 days:
```
Comment (orlandobrent) [inline README.md:46]: Missed a couple of these...

**Review by orlandobrent — COMMENTED** [review]
There's still a few mentions of 646 in the README

File: README.md (hunk 1 above)
Comment (cpeskin12) [reply README.md:46]: argh, good catch!

Comment (cpeskin12) [reply README.md:46]: @orlandobrent - should be taken care of now on master

**Pull request merged**